setup.py

```diff
@@ -0,0 +1,50 @@
+import os
+from setuptools import setup
+
+# read in README
+this_dir = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(this_dir, 'README.md'), 'rb') as f:
+    long_description = f.read().decode().strip()
+
+# requirements
+install_requires = [
+    "hop-client >= 0.0.5",
+    "pygcn"
+]
+extras_require = {
+    'dev': ['pytest', 'pytest-console-scripts', 'pytest-cov', 'flake8', 'flake8-black'],
+}
+
+setup(
+    name = 'gcn2hop',
+    url = 'https://github.com/scimma/gcn2hop',
+    author = 'Ron Tabia',
+    license = 'GNU GENERAL PUBLIC',
+    packages = ['gcn2hop'],
+    entry_points = {
+        'console_scripts': [
+            'gcn2hop = gcn2hop.gcn2hop:main',
+        ],
+    },
+
+    python_requires = '>=3.6.*',
+    install_requires = install_requires,
+    extras_require = extras_require,
+
+    classifiers = [
+        'Development Status :: 2 - Pre-Alpha',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Intended Audience :: Science/Research',
+        'Intended Audience :: Developers',
+        'Natural Language :: English',
+        'Topic :: Scientific/Engineering',
+        'Topic :: Scientific/Engineering :: Astronomy',
+        'Topic :: Scientific/Engineering :: Physics',
+        'Operating System :: POSIX',
+        'Operating System :: Unix',
+        'Operating System :: MacOS',
+        'License :: OSI Approved :: GNU General Public License',
```

Comment:
Change to `License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)`